src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/SignatureVerification/Model/DCCSigningCertificate.swift

```diff
@@ -5,6 +5,6 @@
 import Foundation
 
 public struct DCCSigningCertificate {
-    let kid: Data
-    let data: Data
+    public let kid: Data
```

Comment:
why did you changed that to public?

Reply:
Because otherwise the struct initializer isnt available